setup.py

```diff
@@ -25,7 +25,10 @@
     "python-gitlab>=1.3.0",
     "dparse>=0.4",
     "safety",
-    "jinja2>=2.3"
+    "jinja2>=2.3",
+    "GitPython>=2.1.11",
+    "stashy",
+    "requests-toolbelt"
```

Comment:
Not sure about this dependency. It would be good to reduce dependencies as much as possible.

Do you think there are any other parts of our code that could benefit from using this?